README.md

```diff
@@ -14,7 +14,36 @@ This is a monorepo containing the webapps and libraries in Polymath. In each pac
 # Setup
 
 In the root of the repository run:
-`yarn install`
+
+```
+yarn install
+```
+
+## Installing MongoDB
+
+You will need to run MongoDB in your machine to run the dApps locally
+
+Installing mongodb through with [Homebrew](https://brew.sh/)
```

Comment:
This only works in OS X, in mac you'll just do `apt install mongodb`

Reply:
@josepmc Yeah, I guess I should've mentioned this are OSX-specific install instructions